src/auth/strategies/microsoft.strategy.ts

```
@@ -0,0 +1,73 @@
+    };
+
+    // You can fetch more user info from Microsoft Graph API here if needed
+    // For now, we'll create a basic user profile
+    // Using any here is acceptable as Microsoft profile structure varies
+    // eslint-disable-next-line @typescript-eslint/no-explicit-any
+    const profileObj = profile as Record<string, any>;
+    const user = {
+      email: profileObj.email || profileObj.upn || 'unknown@email.com',
+      firstName:
+        profileObj.givenName ||
+        (profileObj.name && profileObj.name.givenName) ||
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)